libraries/shared/src/shared/Storage.h

```
@@ -69,6 +73,7 @@ namespace storage {
     public:
         ViewStorage(const storage::StoragePointer& owner, size_t size, const uint8_t* data);
         const uint8_t* data() const override { return _data; }
+        uint8_t* mutableData() override { return 0; }
```

this should throw an exception, not return a nullptr.